crypto/src/hash/rpo/mod.rs

```diff
@@ -0,0 +1,912 @@
+use super::{exp_acc, ElementHasher, HashFn};
+use core::{convert::TryInto, ops::Range};
+use winterfell::math::{fields::f64::BaseElement, FieldElement, StarkField};
```

Comment:
I would import this from `crate` and use `Felt` instead of `BaseElement` everywhere.